src/Joonasw.AspNetCore.SecurityHeaders/Csp/Options/CspOptions.cs

```diff
@@ -78,8 +78,12 @@ public class CspOptions
         /// The URL where violation reports should be sent.
         /// </summary>
         public string ReportUri { get; set; }
+		/// <summary>
+		/// If true, the X-Header will be included in the rsponse.
```

Comment:
Fix the indentation. Also the text could be: "Include the X-Content-Security-Policy header in responses".